geodiff/src/drivers/sqlitedriver.cpp

```diff
@@ -8,9 +8,81 @@
 #include "changesetreader.h"
 #include "changesetwriter.h"
 #include "changesetutils.h"
+#include "geodifflogger.hpp"
 
 #include <memory.h>
 
+
+static void logApplyConflict( const std::string &type, const ChangesetEntry &entry )
```

Comment:
can we just rename it logApplySqliteConflict & logApplyPgConflict? (two static function in 2 cpp files?) also maybe in the warn string, add "PG" and "SQLITE" keyword so in logs it is obvious where it came from?